app/database_etl/postgres_tables_handler/table_formatter.py

```diff
@@ -29,6 +32,11 @@ def add_mapped_variables(df):
         else:
             gbd_region_col.append(None)
             gbd_subregion_col.append(None)
+
+            # Send slack message and log warning
```

Comment:
Good catch here - are there other mappings that we could do something like this for? It'd be great to have this in place for the other mappings in the ETL e.g., LMIC/HIC, general population / special populations, and so on.